app/services/sso/jwt.rb

```diff
@@ -35,5 +35,10 @@ def jwt_token_from_request
     def bearer_token_set?
       request.authorization.present? && request.authorization.start_with?('Bearer')
     end
+
+    def valid_issuer?
+      return true unless jwt_token.decoded_payload.key?('iss')
+      false
```

Comment:
Isn't this whole function equal to:
```suggestion
      !jwt_token.decoded_payload.key?('iss')
```